crawl-ref/source/spl-summoning.cc

```diff
@@ -3081,3 +3082,50 @@ spret foxfire_swarm()
     canned_msg(MSG_NOTHING_HAPPENS);
     return spret::fail; // don't spend piety, do spend a turn
 }
+
+bool summon_a_spider(const actor &agent, coord_def pos, god_type god,
```

Comment:
Minor: think we'd more usually call this just `summon_spider`. We don't usually bother with articles in our function names.